src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1372,6 +1372,9 @@ I agree to all of the foregoing:</value>
   <data name="Some_color_scheme_colors_are_being_overridden_by_changes_made_in_the_Environment_Fonts_and_Colors_options_page_Choose_Use_Defaults_in_the_Fonts_and_Colors_page_to_revert_all_customizations" xml:space="preserve">
     <value>Some color scheme colors are being overridden by changes made in the Environment &gt; Fonts and Colors options page. Choose `Use Defaults` in the Fonts and Colors page to revert all customizations.</value>
   </data>
+  <data name="Razor_CSharp_Language_Server_Client" xml:space="preserve">
+    <value>Razor C# Language Server Client</value>
```

Comment:
why does this need a localized name?  when is this shown to the user?

Reply:
If an lsp command takes a long time, it pops up a dialog showing this

Reply:
Should we pick a series of words that mean something to our customers then? (I'm not sure if this is a series of words that makes sense to *us*... :smile:)

Reply:
If you have suggestions, I'm all ears.